packages/contracts/Releases.md

```diff
@@ -1,5 +1,29 @@
 # Contract Releases
 
+Commit: [f8aee0a0fa7ae5825130b9726a18ef6e36ddc121](https://github.com/aragon/core/commit/f8aee0a0fa7ae5825130b9726a18ef6e36ddc121)
```

Comment:
Can we update the script that generates this?  `core` should now be `osx` in the URL

Reply:
done